### DISCUSSION TITLE
[CodePipeline] Insufficient permissions while pulling source from GitHub but working for new Pipeline

### DISCUSSION THREAD

**Discussion by UntilTech** [aws/aws-cdk#23668] [Q&A]
Hello,

My Github Auth Token expired yesterday. So my pipeline couldn't pull the source, and failed at the first step.

Obviously I updated the token, and I updated the token in Secret Manager (where I keep it)...but the Pipeline seems to have cached the previous value of the token.

I think caching is the issue as deploying a new pipeline using the exact same GitHub token from Secret Manager works like a charm.

Is that a normal behaviour ?

Kind Regards,

Chris

**Comment by Gutyn** [comment]
@UntilTech did you figure this one out? thank you.
